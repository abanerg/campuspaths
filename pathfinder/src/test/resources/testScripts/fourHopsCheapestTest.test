# Create graph

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E

AddEdge graph A B 1.000
AddEdge graph A E 10.000
AddEdge graph B C 1.000
AddEdge graph C D 1.000
AddEdge graph D E 1.000

# Should be path from A to E with cost 4.000

FindPath graph A E
