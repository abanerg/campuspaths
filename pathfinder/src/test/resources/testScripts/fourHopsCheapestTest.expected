# Create graph

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph

added edge 1.000 from A to B in graph
added edge 10.000 from A to E in graph
added edge 1.000 from B to C in graph
added edge 1.000 from C to D in graph
added edge 1.000 from D to E in graph

# Should be path from A to E with cost 4.000

path from A to E:
A to B with weight 1.000
B to C with weight 1.000
C to D with weight 1.000
D to E with weight 1.000
total cost: 4.000
